service/src/test/java/bio/terra/landingzone/app/StartupInitializer.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.landingzone.app;
+
+import bio.terra.common.migrate.LiquibaseMigrator;
+import bio.terra.landingzone.library.LandingZoneMain;
+import org.springframework.context.ApplicationContext;
+
+public class StartupInitializer {
+  private static final String CHANGELOG_PATH = "landingzonedb/changelog.xml";
+
+  public static void initialize(ApplicationContext applicationContext) {
```

Comment:
This is needed for spring to set up the context correctly, in order to auto-wire components. There is an equivalent class in `testharness`.